wallet/serializers.py

```diff
@@ -16,3 +16,23 @@ class Meta:
             "created_on",
             "modified_on",
         )
+
+    def create(self, validated_data):
+        """
+        1)  Check how many wallets the user has.
+            If the user has 5 or more wallets, then raise Exception
+
+        2)  Add default bonus from bank for each new created wallet:
+            if wallet currency USD or EUR - balance=3.00,
+            if RUB - balance=100.00
+        """
+
+        print(validated_data)
+        if Wallet.objects.filter(user__exact=validated_data.get("user")).count() >= 5:
+            raise Exception("User can't create more than 5 wallets")
+        if validated_data["currency"] in ["USD", "EUR"]:
+            validated_data["balance"] += 3
+        elif validated_data["currency"] == "RUB":
+            validated_data["balance"] += 100
```

Comment:
it's better to create constants in the wallet model for the bank bonuses and max amount of wallets.
something like this 
```
MAX_WALLETS = 5
BANK_BONUS_USD_EUR = 3.00
BANK_BONUS_RUB = 100.00
```
then you will be able to get these value as an attribute of the wallet model.

remove print, please

Reply:
Done